clients/cloud/java-springboot/README.adoc

```diff
@@ -0,0 +1,172 @@
+= Spring Boot with Java Example Confluent Cloud Client
+
+NOTE: Produce messages to and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) using the Spring Boot using Java API.
+
+== Prerequisites
+
+* Java 1.8 or higher to run the demo application
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.clients-ccloud) cluster
+* Initialize a properties file at `src/main/resources/application-ccloud.properties` with configuration to your Confluent Cloud cluster:
```

Comment:
@gAmUssA all the other examples use a standard Java properties file with just the key properties to connect to a cluster.  

```
$ cat $HOME/.ccloud/config
bootstrap.servers=<BROKER ENDPOINT>
ssl.endpoint.identification.algorithm=https
security.protocol=SASL_SSL
sasl.mechanism=PLAIN
sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username\="<API KEY>" password\="<API SECRET>";
```

However, this PR deviates from that.  Can this example follow the other examples?  If Spring needs `.application-ccloud.properties`, then the example should auto-generate it from the standard Java properties file.

Reasons to do it this way:

1. Consistency
2. Eventually Confluent CLI will autogenerate the config file, so this example can just consume it.

Reply:
it's done!